### DISCUSSION TITLE
Problem after adding dependencies for preferences

### DISCUSSION THREAD

**Discussion by danilo576** [mocklets/pluto#205] [Q&A] [answered]
After adding mentioned dependencies I had a problem with the navigation component. Build output ,,cannot find symbol method getBackStack()"

**Comment by srtvprateek** [comment]
created a issue for the bug https://github.com/plutolib/pluto/issues/206
can you share the details of error & screenshot for dependency code there?

**Reply by danilo576** [reply]
![image](https://user-images.githubusercontent.com/61258071/213987209-fd2e3754-2210-4e92-8b0d-7cc6829edb5d.png)

**Comment by danilo576** [comment, marked as answer]
I solved this problem by replacing navController.getBackStack().getLast().getDestination().getId() with **navController.getCurrentBackStackEntry().getDestination().getId()** and everything works as we expect. This lib is really powerful!

**Reply by srtvprateek** [reply]
thanks @danilo576
